fathom/src/surface/elaboration.rs

```diff
@@ -80,207 +80,224 @@ impl<'arena> RigidEnv<'arena> {
         use crate::core::Prim::*;
         use crate::core::Term;
 
+        const VAR0: core::Term<'_> = core::Term::RigidVar(env::LocalVar::last());
+        const UNIVERSE: core::Term<'_> = core::Term::Universe;
+        const FORMAT_TYPE: core::Term<'_> = core::Term::Prim(Prim::FormatType);
+        const BOOL_TYPE: core::Term<'_> = core::Term::Prim(Prim::BoolType);
+        const U8_TYPE: core::Term<'_> = core::Term::Prim(Prim::U8Type);
+        const U16_TYPE: core::Term<'_> = core::Term::Prim(Prim::U16Type);
+        const U32_TYPE: core::Term<'_> = core::Term::Prim(Prim::U32Type);
+        const U64_TYPE: core::Term<'_> = core::Term::Prim(Prim::U64Type);
+        const S8_TYPE: core::Term<'_> = core::Term::Prim(Prim::S8Type);
+        const S16_TYPE: core::Term<'_> = core::Term::Prim(Prim::S16Type);
+        const S32_TYPE: core::Term<'_> = core::Term::Prim(Prim::S32Type);
+        const S64_TYPE: core::Term<'_> = core::Term::Prim(Prim::S64Type);
+        const POS_TYPE: core::Term<'_> = core::Term::Prim(Prim::PosType);
+
         let mut def = RigidEnvBuilder::new(interner, scope);
```

Comment:
Minor: Might be worth renaming the builder now? I originally named it this so that a line like `def.format(FormatU64Le);` read like `define format FormatU64Le` but now define is in the `define_prim` method call. Maybe call it `env` now?

Reply:
Oh good idea